src/hoplon/core.cljs

```diff
@@ -97,9 +97,10 @@
                         (not x) (with-let [ks ks]
                                   (when-not (new? k)
                                     (let [n (->node k)]
-                                      (.call removeChild this n)
-                                      (when-not *preserve-event-handlers*
-                                        (.remove (js/jQuery n))))))
+                                      (when (= this (.-parentNode n))
```

Comment:
Might want to do `(.remove (js/jQuery n))` even if the node has been removed from the DOM already?